### PR TITLE
Rewrite docs

### PR DESCRIPTION
**Pull request by ndharasz**
- Rewrite README
- Delete unnecessary Account webhook image
- Move extraneous docs to Wiki
- delete unnecessary terraform var template file
- add info command to linux script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,65 +3,84 @@
...
-Questions or feedback? Join us at [RocketChat](https://community.numer.ai/channel/compute)
+If you still cannot find a solution or answer, include the following information with your issue/message:
 
-![Architecture Diagram](https://github.com/numerai/numerai-cli/blob/master/docs/compute_architecture.png)
+- The command you ran that caused the error
+- Version from running `pip3 show numerai-cli`
+- System Information from running
+    - Mac: `system_profiler SPSoftwareDataType && system_profiler SPHardwareDataType`
+    - Linux: `lsb_release -a && uname -a`
+    - Windows: `powershell -command "Get-ComputerInfo`
```
Comment (pschork) [inline README.md:25]: `powershell -command "Get-ComputerInfo"`

File: README.md [hunk 2]
```diff
@@ -3,65 +3,84 @@
 [![CircleCI](https://circleci.com/gh/numerai/numerai-cli.svg?style=svg)](https://circleci.com/gh/numerai/numerai-cli)
 [![PyPI](https://img.shields.io/pypi/v/numerai-cli.svg?color=brightgreen)](https://pypi.org/project/numerai-cli/)
 
-This is a CLI for setting up a Numerai compute node and deploying your models to it. This sets up a compute cluster in AWS (Amazon Web Services), and is architected to cost a minimal amount of money to run (on average, you will spend less than $1 per month).
+CLI to set up a Numerai Compute node in AWS (Amazon Web Services) and deploy your models to it. 
+This is architected to cost a minimal amount of money to run (less than $5 per month on average).
```
Comment (pschork) [inline README.md:7]: This solution is architected to cost less and $5/mo on average (actual costs may vary).

File: README.md [hunk 3]
```diff
@@ -3,65 +3,84 @@
...
+It has been tested and found working on MacOS/OSX, Windows 10, and Ubuntu 18.04, 
+but should theoretically work anywhere that docker and Python are available.
```
Comment (pschork) [inline README.md:9]: Python 3 are available

File: README.md [hunk 4]
```diff
@@ -3,65 +3,84 @@
...
-    apt update && apt install -y libcurl4 curl && curl https://raw.githubusercontent.com/numerai/numerai-cli/ndharasz/setup-scripts/scripts/setup-ubu18.sh | bash
+    apt update && apt install -y libcurl4 curl && curl https://raw.githubusercontent.com/numerai/numerai-cli/master/scripts/setup-ubu18.sh | bash
     ```
 
 - Windows Command Prompt (windows key, type `cmd`, select Command Prompt):
     ```
-    powershell -command "$Script = Invoke-WebRequest 'https://raw.githubusercontent.com/numerai/numerai-cli/ndharasz/setup-scripts/scripts/setup-win10.ps1'; $ScriptBlock = [ScriptBlock]::Create($Script.Content); Invoke-Command -ScriptBlock $ScriptBlock"
+    powershell -command "$Script = Invoke-WebRequest 'https://raw.githubusercontent.com/numerai/numerai-cli/master/scripts/setup-win10.ps1'; $ScriptBlock = [ScriptBlock]::Create($Script.Content); Invoke-Command -ScriptBlock $ScriptBlock"
     ```
+If you run into issues running one of these scripts, please report immediately to Rocketchat followig.
```
Comment (pschork) [inline README.md:69]: If you run into issues running one of these scripts, please report your issue in the Rocketchat #support channel

File: README.md [hunk 5]
```diff
@@ -3,65 +3,84 @@
...
 
-See the [Prerequisites Help](#prerequisites-help) section if you need further help getting these setup.
-
-The CLI has been tested and found working on MacOS/OSX, Windows 10, and Ubuntu 18.04, 
-but should theoretically work anywhere that docker and Python are available.
 
-Install this library with:
+After runnning the setup script and it says it found Python and Docker, install this library with:
```
Comment (pschork) [inline README.md:72]: After running the setup script and confirming Python and Docker requirements, install `numerai-cli` via

File: README.md [hunk 6]
```diff
@@ -72,397 +91,65 @@ numerai docker copy-example
...
-
-And then lets look at the Dockerfile:
+Your compute node is now setup and ready to run. It saves the webhook URL in `.numerai/submission_url.txt` that triggers your docker container. 
+You can configure [your Numerai account](https://numer.ai/account) to use this webhook by entering it in the "Compute" section.
+It will be called Saturday morning right after a new round opens, and if your job fails (one ore more models have not submitted successfully) 
+then it will be triggered again around 24 hours later.
 
-```
-FROM python:3
+NOTE: The default example does _not_ stake, so you will still have to manually do that every week.
```
Comment (pschork) [inline README.md:99]: NOTE: The default example does _not_ make stake changes.

**Review by pschork — APPROVED** [review]
Nice doc update! See minor comments

File: README.md (hunk 4 above)
Comment (ndharasz) [reply README.md:69]: We have a #compute channel that is linked above, that was the channel I was going to point them to

**Pull request merged**